sass/style.scss

```diff
@@ -7,115 +7,6 @@
  *   green: #7DD36B
  *   yellow: #F1B136
  */
-
-@import url('https://fonts.googleapis.com/css?family=Overpass:200,300,400,600');
-body {
-    font-size: 62.5%;
-    font-family:'Overpass', sans-serif;
-    font-weight: 200;
-    margin: 0 auto;
-    width: 100%;
-    line-height: 20px;
-    color: #E4E4E4;
-    background-color: #393939;
-}
-h1 {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 0px 0px;
-    width: 163px;
-}
-h2 {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 25px 0px;
-    font-family: 'Overpass', sans-serif;
-    font-size: 4.4em;
-    font-weight: 600;
-    color: #7197FA;
-    line-height: 55px
-}
-h3 {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 0px 0px;
-    font-family: 'Roboto', sans-serif;
-    font-size: 1.8em;
-    font-weight: 400;
-}
-p {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 0px 0px;
-    font-size: 1.3em;
-    margin-bottom: 20px
-}
-ul {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 0px 0px;
-    list-style-type: none;
-}
-li {
-    padding: 0px 0px 0px 0px;
-    margin: 0px 0px 0px 0px;
-    font-size: 1.3em;
-    list-style: none;
-}
-a {
-    text-decoration: none;
-}
-img {
-    width: 100%;
-    height: auto;
-}
-.row {
-    display: flex;
-    flex-flow: row;
-    justify-content: center;
-    align-items: center;
-}
-.row-wrap {
-    display: flex;
-    flex-flow: row wrap;
-    justify-content: space-around;
-    margin-top: 70px;
-}
-.col {
-    display: flex;
-    flex-flow: column;
-    justify-content: center;
-}
-header {
-    margin-bottom: 70px;
-    background-color: #DE3352;
-    height: 281px;
-}
-header img {
-    width: 150px;
-    height: 150px;
-}
-header p {
-    width: 300px;
-    font-size: 1.8em;
-    margin-left: 35px;
-    text-shadow: 0px 5px 3px black;
-}
-main section {
-    display: flex;
-    flex-flow: column;
-    align-items: center;
-    width: 90%;
-    margin: 0 auto;
-    text-align: center;
-    padding: 35px 0px 35px 0px;
-}
-main section p {
-    max-width: 600px;
-    min-width: 300px;
-}
-main section:last-child img {
-    width: 300px;
-    margin-right: 70px;
-}
-main section:last-child h2 {
-    margin-bottom: 70px;
-}
-.contact {
-    text-align: left;
-}
+@import 'var';
+@import 'class';
+@import 'elements';
```

Comment:
Another thing you can look for to make use of Sass is to use mixins to replace blocks of code you enter often. Consider this as an example:

```scss
@mixin reset() {
  margin: 0;
  padding: 0;
}
```

Then you'd use it like this:

```scss
h1 {
  @include reset;
  width: 163px;
}
```

And the compiled output would be:

```css
h1 {
  margin: 0;
  padding: 0;
  width: 163px;
}
```

So that's a really simple example, so see if there are any other places you can do this, esp if you involve [input parameters](http://sass-lang.com/guide#topic-6) that allow you to pass values into the mixin.